lib/modules/manager/kotlin-script/index.ts

```diff
@@ -0,0 +1,11 @@
+import { MavenDatasource } from '../../datasource/maven';
+import * as mavenVersioning from '../../versioning/maven';
+
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  fileMatch: ['^.*\\.main\\.kts$'],
+  versioning: mavenVersioning.id,
```

Comment:
If the datasource is always maven then I don't think this is necessary

Reply:
What should I do instead?

Reply:
The line can be removed

Reply:
Ah right, Maven datasource contains Maven versioning already.